examples/concepts/javabip/casinoBroken/Constants.java

```diff
@@ -47,7 +47,6 @@ public class Constants {
     public static final String OUTGOING_FUNDS = "OUTGOING_FUNDS";
     public static final String INCOMING_FUNDS = "INCOMING_FUNDS";
     public static final String AVAILABLE_FUNDS = "AVAILABLE_FUNDS";
-    public static final String CASINO = "CASINO";
```

Comment:
There is already a `CASINO` constant above, right? Not sure why I added this one...

Reply:
Indeed works fine without the extra public static final